directord/drivers/msg.proto

```diff
@@ -0,0 +1,77 @@
+// generate files
```

Comment:
this may need to move into a lib folder within the drivers?

Reply:
it can go in the generated folder if we want. just needs to live somewhere